scripts/Makefile

```diff
@@ -82,4 +82,21 @@ fortune: check-core-folders minio
 	$(MAKE) fortune-recording-oracle & PID_RECO=$$!; \
 	$(MAKE) reputation-oracle & PID_REPO=$$!; \
 	$(MAKE) subgraph & \
-	wait
\ No newline at end of file
+	wait
+
+web3-subgraph:
```

Comment:
The difference is the database config. `subgraph` is using docker compose file from fortune directory, which has db setup with other databases like reputation/recording oracles.
However, this docker compose inside `web3` uses db docker dependency with graph-node-db database only. In addition it exposes 5433 port from the postgres container, so that we won't have port conflicts while we run postgres container for other services.

Reply:
Ideally, fortune script should be reconfigured, so that it only has docker compose for fortune services, and use web3 scripts to run web3 environment - hardhat local node, deploy contracts, and run subgraph.